.github/workflows/test.yaml

```diff
@@ -16,6 +16,12 @@ jobs:
       GOROOT: ""
       # XXX: only skip format checks on 1.13+
       SKIP_GOFMT: 1
+      # XXX: compat env for "check-pr-title.py" in "run-checks", can go
```

Comment:
this sentence is a bit hard to read, would propose this instead:

```
      # XXX: compat env for "check-pr-title.py" in "run-checks", this can go
      #          away once we switch away from that script. Note that we cannot currently

```